src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -1,12 +1,328 @@
+        channel_ids = [str(1 + i) for i in range(num_channels)]
+        binary_file_path = self.folder_path / f"{self.session}.{self.binary_file_extension}"
+        if binary_file_path.is_file():
+            from spikeinterface.core.binaryrecordingextractor import (
+                BinaryRecordingExtractor,
+            )
+
+            self.recording_extractor = BinaryRecordingExtractor(
+                file_paths=[binary_file_path],
+                sampling_frequency=sampling_frequency,
+                num_chan=num_channels,
+                dtype=dtype,
+                t_starts=None,
+                file_offset=0,
+                gain_to_uV=gains_to_uv,
+                offset_to_uV=None,
+                channel_ids=channel_ids,
+            )
+        else:
+            from spikeinterface.core.numpyextractors import NumpyRecording
+
+            traces_list = [np.empty(shape=(1, num_channels))]
+            dummy_recording = NumpyRecording(
+                traces_list=traces_list,
+                sampling_frequency=sampling_frequency,
+                channel_ids=channel_ids,
+            )
+
+            self.recording_extractor = dummy_recording
+            self.recording_extractor.set_channel_gains(channel_ids=channel_ids, gains=np.ones(num_channels) * gain)
+
+        self.recording_extractor = add_channel_metadata_to_recorder_from_session_file(
+            recording_extractor=self.recording_extractor, session_path=self.folder_path
+        )
+
+        self.recording_extractor = add_channel_metadata_to_recorder_from_channel_map_file(
+            recording_extractor=self.recording_extractor, session_path=self.folder_path
+        )
+
+    def get_original_timestamps(self):
+        num_frames = self.recording_extractor.get_num_frames()
+        sampling_frequency = self.recording_extractor.get_sampling_frequency()
+        timestamps = np.arange(num_frames) / sampling_frequency
+        return timestamps
+
+
+class CellExplorerLFPInterface(CellExplorerRecordingInterface):
+    keywords = BaseRecordingExtractorInterface.keywords + [
+        "extracellular electrophysiology",
+        "LFP",
+        "local field potential",
+        "LF",
+    ]
+
+    sampling_frequency_key = "srLfp"
+    binary_file_extension = "lfp"
+
+    def __init__(self, folder_path: FolderPathType, verbose=True, es_key: str = "ElectricalSeriesLFP"):
+        super().__init__(folder_path, verbose, es_key)
+
+    def add_to_nwbfile(
+        self,
+        nwbfile: NWBFile,
+        metadata: Optional[dict] = None,
+        stub_test: bool = False,
+        starting_time: Optional[float] = None,
+        write_as: Literal["raw", "lfp", "processed"] = "lfp",
+        write_electrical_series: bool = True,
+        compression: Optional[str] = None,
+        compression_opts: Optional[int] = None,
+        iterator_type: str = "v2",
+        iterator_opts: Optional[dict] = None,
+    ):
+        return super().add_to_nwbfile(
```

Comment:
```suggestion
        super().add_to_nwbfile(
```

No need for `return` here since `add_to_nwbfile` returns `None`

Reply:
If all the others are doing this that's fine but I feel it should return and nwbfile. I have not answered to your replies in the pipeiline interface because of lack of time.